pkg/service/v0/errorcode/errorcode.go

```diff
@@ -0,0 +1,75 @@
+package errorcode
+
+import (
+	"net/http"
+
+	"github.com/go-chi/render"
+	msgraph "github.com/yaegashi/msgraph.go/v1.0"
+)
+
+// ErrorCode defines code as used in MS Graph - see https://docs.microsoft.com/en-us/graph/errors?context=graph%2Fapi%2F1.0&view=graph-rest-1.0
+type ErrorCode int
+
+const (
+	// AccessDenied : The caller doesn't have permission to perform the action.
```

Comment:
This kind of notation is not used anywhere for godoc comments. Drop the colon, and write something like `// AccessDenied defines the error if the caller...`